### DISCUSSION TITLE
Is there a health check endpoint for Seq itself?

### DISCUSSION THREAD

**Discussion by dlaudams** [datalust/seq-tickets#1251] [Help] [answered]
Does Seq have a specific health check endpoint that can be configured for monitoring of Seq itself, such as by a container or by a load balancer?

e.g. /health

**Comment by nblumhardt** [comment, marked as answer]
Hi Daniel,

Thanks for getting in touch.

Current released versions tend to use `/api` for this, which works fine (returns 200 when Seq is available, and 5xx otherwise)..

In 2021.3 (due in the near future), we've introduced a dedicated `/health` endpoint:

https://github.com/datalust/seq-tickets/issues/1200

Both will return 200 when Seq is available, but `/health` has a more suitable response body, and will report the current node's health in multi-node scenarios (rather than the leader node's). Just letting you know in case you want to switch over to it in future.

Hope this helps!
Nick

**Reply by dlaudams** [reply]
Perfect, thanks!
